source/Calamari.Common/Features/Processes/Semaphores/LockFileBasedSemaphore.cs

```diff
@@ -8,11 +8,11 @@ namespace Calamari.Common.Features.Processes.Semaphores
 {
     public class LockFileBasedSemaphore : ISemaphore
     {
-        public enum AquireLockAction
+        public enum AcquireLockAction
```

Comment:
I'm glad to see this fixed, but given the size of this PR, I would prefer to see things like this done separately.